muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/static/iframe.tsx

```diff
@@ -147,7 +170,6 @@ export default class Iframe extends React.Component<
             delete iframeProps.width;
             containerStyle = {
               ...containerStyle,
-              paddingTop: "56.25%",
```

Comment:
Removal of this will effectively break 16:9 ratio which can be seen when embedding 16:9 video from youtube.

With `padding-top:  56.25%` the video dimensions are as follows: `width: 658px` and `height: 349px`

When comparing this to devel version, same video that has correct 16:9 aspect ratio has following dimensions: `width: 658px` and  `height: 370px`.

Of course iframe width is always 100% within materials as we process it. Padding-top rule is meant to preserve correct 16:9 aspect ratio of videos when width is artificially adjusted to full width of 100%.

This means that the video will be flattened a bit and therefore makes video fall out of aspect ratio regardless whether the video has black bars or not.

Current solution to add extra element to work as an activating element would mean that it has to inherit it's parent elements styles which means that `.material-page__iframe-wrapper` element could be unnecessary altogether. 

I think solution to this problem is that we either use existing `.material-page__iframe-wrapper` element to function as an state activating element or we transform styles from that element to the `.click-outside-listener ` element and remove the `.material-page__iframe-wrapper` element.